db/seeds.rb

```diff
@@ -12,14 +12,31 @@
 
 Rails.logger.debug "Personas successfully created!"
 
+Money.locale_backend = nil
+
+regions = [
+  { name: "Inner London", claims_funding_available_per_hour: 53.60 },
+  { name: "Outer London", claims_funding_available_per_hour: 48.25 },
+  { name: "Fringe", claims_funding_available_per_hour: 45.10 },
+  { name: "Rest of England", claims_funding_available_per_hour: 43.18 },
```

Comment:
I know these can be implicit, but would it make it clearer in the tests that these are money values if we passed through money objects?

```suggestion
  { name: "Inner London", claims_funding_available_per_hour: Money.from_amount(53.60, "GBP") },
  { name: "Outer London", claims_funding_available_per_hour: Money.from_amount(48.25, "GBP" },
  { name: "Fringe", claims_funding_available_per_hour: Money.from_amount(45.10, "GBP") },
  { name: "Rest of England", claims_funding_available_per_hour: Money.from_amount(43.18, "GBP") },
```